text/0000-security-team.md

```diff
@@ -0,0 +1,181 @@
+
+_In the example case, a vulnerability is disclosed to the disclosure contacts listed in Grin's security policy. It is shown that the community gains greater transparency into the decision making process and related outcomes with the proposed security team. Additionally the disclosure map is more tightly contained in the proposed process, resulting in less risk of vulnerability exploitation to the community._
+
+### Current Process
+
+* Grin's security contacts make initial assessment of the validity of the vulnerability disclosure
+* If disclosure determined to be valid, all details of the disclosure are shared with the entire core team
+* With a conversation between the entire core team, a plan of action is determined and executed, following Grin's responsible disclosure policy
+* Once the vulnerability is mitigated the core team may optionally share some of the details with the community
+
+### Proposed Process
+* Grin's security contacts, all members of the security team, make initial assessment of the validity of the vulnerability disclosure
+* If disclosure determined to be valid, a conversation and plan of action is determined and executed by the security team, following Grin's responsible disclosure policy
+* Once the vulnerability is mitigated the security team must disclose all information that is not still security-critical with the community
+
+# Reference-level explanation
+[reference-level-explanation]: #reference-level-explanation
+
+Grin's security team will be implemented as stated below.
+
+## Membership
+
+* Bootstrapped by existing security contacts (@lehnberg, @hashmap, @joltz)
+* Minimum three, maximum five members
+* Membership is recommended by the security team and approved by the core team
+* There must be at least one well-represented core developer on the team, chosen by the core team
+* All members must have relevant domain expertise in security, cryptography, the Grin codebase or Grin community
+* All members must have high competency in maintaining personal operational security (keeping private keys secure, maintaining secure communications channels etc.)
+* Membership is revoked by failure to update the canary, unanimous decision by team or decision from the core team
+  * Note if the core team engages in removing members from the security team without reasonable justification, the security team loses purpose, becomes ineffective and is relegated to useless security theater
+
+### Core Representation
+
+The core team is responsible for choosing one developer who they trust to represent the core interests in the security team. This ensures that the core team has adequate representation in security related decisions, without overpowering the majority of discussion. Additionally, with the veto rules defined below, we ensure that in cases of disagreement the security team cannot make an impactful decision without the core team's awareness and input.
```

Comment:
Similarly, this entire section might then be removed.

Reply:
Likewise, removed this section as well.